plugins/database/src/db_plugin.cpp

```diff
@@ -7839,7 +7839,8 @@ irods::error db_mod_user_op(
         int i;
         char userIdStr[MAX_NAME_LEN];
         i = decodePw( _ctx.comm(), _new_value, decoded );
-        if (strlen(decoded) > MAX_PASSWORD_LEN - 8) {
+        if (i == CAT_PASSWORD_ENCODING_ERROR || strlen(decoded) > MAX_PASSWORD_LEN - 8) {
```

Comment:
Is there ever a case where `CAT_PASSWORD_ENCODING_ERROR` is returned but it's not because the password is too long? What if the password is too short (<3 characters)?

Reply:
We discussed this in standup the other day.  It is hard to say due to all the stuff that is going on in decodePw.  That possibility is why we decided to not update decodePw but instead catch the error here.

Reply:
@JustinKyleJames - I still think a comment would be helpful here, but if we're not going to do that, please resolve this conversation. Thanks!

Reply:
Agreed.

@JustinKyleJames If you've acquired knowledge about this, let's write it down for the next person. This just helps us move faster.

Reply:
I added a comment for this.  I said that the CAT_PASSWORD_ENCODING_ERROR occurs when the password is not of the correct length.  There may be other cases where a CAT_PASSWORD_ENCODING_ERROR isn't due to that but I am not able to determine from the code if that is the case.